### DISCUSSION TITLE
Styling Calendar

### DISCUSSION THREAD

**Discussion by kkpongza** [mantinedev/mantine#6395] [Q&A] [answered]
i have a calendar component from mantine v6 and i want to set the width to fill the screen but the calendar component not expand
![image](https://github.com/mantinedev/mantine/assets/120788226/936d839f-b7fa-4f0a-9c33-c7df0284bd2f)

<div style={{display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center', backgroundColor: '#edf0f2', borderRadius: '12px', padding: '5px'}}>
            <CustomCalendarHeader date={date} setDate={setDate} />

            <Calendar
                date={date}
                onDateChange={setDate}
                hideOutsideDates
                size= {isLargerThanSm ? 'xl': 'lg'}
                
                getDayProps={(date) => {
                  const isSelected = selectedDate && dayjs(date).isSame(selectedDate, 'date');
                  const isToday = dayjs(date).isSame(dayjs(), 'date');
                  // Styling for today
                  let style: Style = { color: 'black' };
                  
                  if (isToday) {
                    style = { backgroundColor: '#d9dadb', color: 'black', borderRadius: '50%'};
                  }

                  // Styling for selected dates
                  if (isSelected && !isToday) {
                    style = { backgroundColor: '#4C6EF5', color: 'white', borderRadius: '50%' };
                  }

                  return {
                    style,
                    onClick: () => handleSelect(date),
                  };
                }}

                styles={{
                  calendar: {display: 'grid', },
                  calendarHeader: { display: 'none' },  // Hides the calendar header
                }}
                style={{marginTop: '20px'}}
              />
          </div>
          
          you will see that the date border is out of the background because i cannot handle the size of the calendar
          
![image](https://github.com/mantinedev/mantine/assets/120788226/47890b70-7835-4c98-b47a-f936ad6fab7e)

pls helpppppp

**Comment by rtivital** [comment]
You can apply styles to components with Styles API – https://v6.mantine.dev/styles/styles-api/

**Reply by kkpongza** [reply]
how ;-;

**Reply by kkpongza** [reply, marked as answer]
i got it by set the week day width to 14.2 (100/7)
